modules/ROOT/pages/dw-core-functions-sizeof.adoc

```diff
@@ -249,3 +249,9 @@ output application/json
 }
 ----
 
+
+[[sizeof5]]
+== sizeOf&#40;n: Null&#41;: Null
```

Comment:
TODO: maybe something other than`n` as param name for these. Check for this elsewhere. See if there's a way to conform with something better.